summarizer/model_services/summary_service.py

```diff
@@ -2,9 +2,9 @@
 sys.path.append("src")
 import gradio as gr
 from llama_cpp import Llama
-from chat import clip_history, chunk_tokens
+from llamacpp_utils import clip_history, chunk_tokens
  
-llm = Llama("llama-2-7b-chat.Q5_K_S.gguf",
+llm = Llama("models/llama-2-7b-chat.Q5_K_S.gguf",
```

Comment:
Change does not seem required to me as the image does not start anymore because the model cannot be found